### DISCUSSION TITLE
STM32SD library: File does not inherit from Stream

### DISCUSSION THREAD

**Discussion by pschatzmann** [stm32duino/Arduino_Core_STM32#2428] [Q&A] [answered]
I noticed that for the File class in your STM32SD library you have all the methods available, but you do not inherit from Arduino Stream.
All other SD libraries I know of (e.g. [SD](https://github.com/arduino-libraries/SD/blob/master/src/SD.h)) do this and unfortunately this prevent me from using your library.

Is there any reason for this and would you accept a pull request to have this implemented ?

**Comment by fpistm** [comment]
Hi @pschatzmann 
All contributions are welcome.

**Reply by fpistm** [reply, marked as answer]
https://github.com/stm32duino/STM32SD/pull/72
